proxy-node-gateway/src/main/java/uk/gov/ida/notification/resources/EidasAuthnRequestResourceV2.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.ida.notification.resources;
+
+import io.dropwizard.jersey.sessions.Session;
+import io.dropwizard.views.View;
+import org.apache.commons.lang.StringUtils;
+import org.opensaml.saml.saml2.ecp.RelayState;
+import uk.gov.ida.notification.EidasSamlParserService;
+import uk.gov.ida.notification.SamlFormViewBuilder;
+import uk.gov.ida.notification.VSPService;
+import uk.gov.ida.notification.dto.EidasSamlParserRequest;
+import uk.gov.ida.notification.dto.EidasSamlParserResponse;
+import uk.gov.ida.notification.saml.SamlFormMessageType;
+import uk.gov.ida.notification.dto.VSPAuthnRequestResponse;
+import uk.gov.ida.notification.views.SamlFormView;
+
+import javax.servlet.http.HttpSession;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.net.URI;
+import java.util.logging.Logger;
+
+@Path("/SAML2/SSO")
+public class EidasAuthnRequestResourceV2 {
+
+    private final Logger log = Logger.getLogger(getClass().getName());
+    public static final String SESSION_KEY_EIDAS_REQUEST_ID = "eidas_request_id";
+    public static final String SESSION_KEY_EIDAS_CONNECTOR_PUBLIC_KEY = "eidas_connector_public_key";
+    public static final String SESSION_KEY_EIDAS_DESTINATION = "eidas_destination";
+    public static final String SESSION_KEY_HUB_REQUEST_ID = "hub_request_id";
+    public static final String SUBMIT_BUTTON_TEXT = "Post Verify Authn Request to Hub";
```

Comment:
If we need to display a page with a button for the users to click that needs to be designed by the content designers. Better still if we could host that page in the hub frontend instead of the gateway - though all that can be done/decided later when we have the new infrastructure in place.